x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -125,175 +129,177 @@ export function XYConfigPanel(props: VisualizationProps<State>) {
 
   return (
     <EuiForm className="lnsConfigPanel">
-      <EuiFormRow>
-        <EuiPopover
-          id="lnsXY_chartConfig"
-          isOpen={isChartOptionsOpen}
-          closePopover={() => setIsChartOptionsOpen(false)}
-          button={
-            <EuiFormRow>
-              <>
-                <EuiButton
-                  iconType="gear"
-                  size="s"
-                  data-test-subj="lnsXY_chart_settings"
-                  onClick={() => setIsChartOptionsOpen(true)}
-                >
-                  <FormattedMessage
-                    id="xpack.lens.xyChart.chartSettings"
-                    defaultMessage="Chart Settings"
-                  />
-                </EuiButton>
-              </>
-            </EuiFormRow>
-          }
-        >
-          <>
-            <EuiFormRow>
-              <EuiSwitch
-                label={i18n.translate('xpack.lens.xyChart.isHorizontalSwitch', {
-                  defaultMessage: 'Rotate chart 90º',
-                })}
-                checked={state.isHorizontal}
-                onChange={() => {
-                  setState({
-                    ...state,
-                    isHorizontal: !state.isHorizontal,
-                  });
-                }}
-                data-test-subj="lnsXY_chart_horizontal"
-              />
-            </EuiFormRow>
-          </>
-        </EuiPopover>
-      </EuiFormRow>
+      <EuiPopover
+        id="lnsXY_chartConfig"
+        isOpen={isChartOptionsOpen}
+        closePopover={() => setIsChartOptionsOpen(false)}
+        button={
+          <EuiButtonIcon
+            iconType="gear"
+            size="s"
+            data-test-subj="lnsXY_chart_settings"
+            onClick={() => setIsChartOptionsOpen(isChartOptionsOpen ? false : true)}
```

Comment:
This should just be `setIsChartOptionsOpen(!isChartOptionsOpen)`

Reply:
```suggestion
            onClick={() => setIsChartOptionsOpen(!isChartOptionsOpen)}
```